registration/auth_urls.py

```diff
@@ -31,9 +31,15 @@
 from django.contrib.auth import views as auth_views
 
 
+def login_user(request, template_name):
+    if request.method == 'POST' and not 'remember_me' in request.POST:
+        request.session.set_expiry(0)
```

Comment:
does the django auth view automatically handle if this parameter is set? Or should a line setting the cookie be placed? (i.e. https://github.com/pennersr/django-allauth/blob/cf45cc605a3d2b49efb1309960df7ee6059175cd/allauth/account/forms.py#L159)